go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/fsnotify/fsnotify v1.7.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/miekg/dns v1.1.62
-	github.com/prometheus/client_golang v1.20.4
+	github.com/prometheus/client_golang v1.20.5
```

Comment:
Falls back to the function used for the prometheus test.
There's nothing for Kmesh.